apps/dotcom/src/app/pages/faq/Accordion.tsx

```diff
@@ -0,0 +1,66 @@
+// Mostly copied from src/app/components/Accordion.tsx
```

Comment:
❓ If it's mostly copied, why not just modify that Accordion to support this use case? What are the major differences?